tests/adapters/test_async_proxy.py

```diff
@@ -376,5 +376,5 @@ async def test_adapter_counter_get_single_node(self, async_proxy_adapter_fixture
         response = await async_proxy_adapter_fixture.adapter.get(path, async_proxy_adapter_fixture.request)
         access_counts = [server.get_access_count() for server in async_proxy_adapter_fixture.test_servers]
 
-        assert path in response.data
+        # assert path in response.data
```

Comment:
Did you want this check?

Reply:
No - as the path is no longer the top-level key in the response. A better check at this point it to match the keys in response.data with those in the param tree of the ProxyTestHandler.